README.md

```diff
@@ -74,6 +75,12 @@ services.AddBlazorQuery();
 
 
 ## Change log
+- **Version 0.0.2 (Sybaris) - 2020-07-29 https://github.com/sybaris/BlazorQuery **
```

Comment:
I don't think it makes sense to put Contributor names on a Version per Version basis. How about a Contributor Section instead?

Reply:
Also no reason to put the URL of the Repo in the Change Log :)

Reply:
Done. I add a section contributors